api-client/src/maps/index.ts

```diff
@@ -60,6 +64,12 @@ export const mapConfig: MapConfigProps = {
 }
 
 export const defaultViewportConfig: MapViewViewport = {
+  latitude: 48.366512,
+  longitude: 10.894446,
+  zoom: 8
+}
+
+export const defaultMercatorViewportConfig: MapViewMercatorViewport = {
   width: 400,
   height: 400,
   latitude: 48.366512,
```

Comment:
```suggestion
  ...defaultViewportConfig
```
Is this Augsburg? Are the coordinates ever used?